### ISSUE TITLE
Bug: schema isn't passed to sqlalchemy_dataset.py during suite scaffolding creation

### ISSUE THREAD

**Issue by lukedyer-peak** [fivetran/great_expectations#2292]
**Describe the bug**
The custom schema isn't passed to `sqlalchemy_dataset.py` during scaffolding of a new expectation suite.  I'm trying to use the new feature introduced in #2244 to define the datasource as the schema and table in the CLI.  Looking through the code the 
`data_asset_name, batch_kwargs` are returned correctly by the CLI https://github.com/great-expectations/great_expectations/blob/76b6e4d122f9e3126aca60d3a3c44309f0b24f95/great_expectations/cli/datasource.py#L1365
```python
data_asset_name, batch_kwargs = transform.prices, {'table': 'prices', 'schema': 'transform', 'data_asset_name': 'transform.prices', 'datasource': '****'}
``` 
However, by the time this is passed to the `SqlAlchemyDataset` the `query_schema=None` here https://github.com/great-expectations/great_expectations/blob/76b6e4d122f9e3126aca60d3a3c44309f0b24f95/great_expectations/dataset/sqlalchemy_dataset.py#L586 instead of the `schema` provided.  So if the table you are wanting to scaffold doesn't live on the default search path then the following exception is raised `sqlalchemy.exc.NoSuchTableError: prices`

**To Reproduce**
On the CLI try to select a schema that isn't the default one - for me this is the `transform` schema

1. `great_expectations suite scaffold transform.prices.warning`
2. Choose `1`
3. Schema name: `transform`
4. Table: `prices`
5. The `sqlalchemy.exc.NoSuchTableError: prices` is raised as `table_name=price` and `query_schema=None` here https://github.com/great-expectations/great_expectations/blob/76b6e4d122f9e3126aca60d3a3c44309f0b24f95/great_expectations/dataset/sqlalchemy_dataset.py#L586

Using the old behaviour of supplying the query `select * from transform.prices` at the same point https://github.com/great-expectations/great_expectations/blob/76b6e4d122f9e3126aca60d3a3c44309f0b24f95/great_expectations/dataset/sqlalchemy_dataset.py#L586 gives `table_name=ge_tmp_7d2ff740` and `query_schema=None`.  This seems to suggest that either the `query_schema` should be set correctly or the temporary table isn't made.

**Expected behavior**
`query_schema` should be `transform` or a temporary table should have been created

**Environment (please complete the following information):**
 - Operating System: MacOS
 - Great Expectations Version: 0.13.5
 - Python: 3.7.9

**Additional context**
None

**Comment by Shinnnyshinshin** [comment]
Hi @lukedyer-peak thank you for your detailed bug report. This bug was addressed in a recent PR ( #2284 ) and the fix should be a part of an upcoming release of 0.13.6.  I'll comment here when the release has actually been made

**Comment by Shinnnyshinshin** [comment]
Hi @lukedyer-peak I wanted to follow up. GE 0.13.6 has been released on PyPi. You should be able to download it at the following link : 

https://pypi.org/project/great-expectations/0.13.6/

**Comment by Shinnnyshinshin** [comment]
please let me know if the issue has been addressed, and if the issue can now be closed

**Comment by lukedyer-peak** [comment]
Hi @Shinnnyshinshin yes the issue has been resolved now.  Thanks for the quick reply.

Closed by #2284
